rust/smoketest/README.md

```diff
@@ -24,7 +25,7 @@ All settings for running the smoke-tests are set via environmental variables. Th
 |TARGET_USERNAME|Username for login in the target during the authenticated scan|empty string|no|Necessary for authenticated scans|
 |TARGET_PASSWORD|Password for login in the target during the authenticated scan|empty string|no|Necessary for authenticated scans|
 |API_KEY|API Key for authenticated communication with `openvasd`|mtls_is_preferred|yes||
```

Comment:
Is the API key mandatory? In the `config.rs` it is optional, and also the default value is not set, so would be None